src/pages/studyView/addChartButton/AddChartButton.tsx

```diff
@@ -484,7 +484,7 @@ class AddChartTabs extends React.Component<IAddChartTabsProps, {}> {
 
     @computed get notificationMessages() {
         let notificationMessages: JSX.Element[] = [];
-        if (this.props.store.customChartSet.size > 0) {
+        if (this.props.store.customChartMap.size > 0) {
```

Comment:
Same here. If the only change you're making to a file is a rename, I'd rather you didn't make it. Sifting through inconsequential refactoring changes in git hitstory is a pain